client/v3/concurrency/session.go

```diff
@@ -124,7 +128,7 @@ func WithTTL(ttl int) SessionOption {
 // This is useful in process restart scenario, for example, to reclaim
 // leadership from an election prior to restart.
 func WithLease(leaseID v3.LeaseID) SessionOption {
-	return func(so *sessionOptions) {
+	return func(so *sessionOptions, logger *zap.Logger) {
```

Comment:
Should I replace `logger` with `_` since this parameter is unused?

Reply:
yes